src/main/java/edu/tseidler/model/BoardParameters.java

```diff
@@ -0,0 +1,25 @@
+package edu.tseidler.model;
+
+class BoardParameters {
+    private final int maxRow;
+    private final int maxCol;
+    private final int winningNumber;
+
+    BoardParameters(int maxRow, int maxCol, int winningNumber) {
+        this.maxRow = maxRow;
+        this.maxCol = maxCol;
+        this.winningNumber = winningNumber;
+    }
+
+    int getMaxRow() {
```

Comment:
What do you think about making fields public in this code implementation?

Reply:
zmieniłem to właśnie z pola publicznego na prywatne z getterem :D